docs/resources/network.md

```diff
@@ -19,7 +19,6 @@ provider "edgecenter" {
 
 resource "edgecenter_network" "network" {
   name       = "network_example"
-  mtu        = 1450
```

Comment:
not needed?

Reply:
on UI we don't have this field and with api call it throws an error about forbidden field. Mtu it is a computed field and generates after creation